fenix/app/src/main/java/org/mozilla/fenix/shopping/middleware/ProductAnalysisMapper.kt

```diff
@@ -32,18 +32,15 @@ private fun ProductAnalysis.toProductReview(isInitialAnalysis: Boolean): Product
         val mappedGrade = grade?.asEnumOrDefault<ReviewQualityCheckState.Grade>()
         val mappedHighlights = highlights?.toHighlights()?.toSortedMap()
 
-        if (mappedGrade == null && mappedRating == null && mappedHighlights == null) {
-            if (isInitialAnalysis) {
-                ProductReviewState.NoAnalysisPresent()
-            } else {
-                ProductReviewState.Error.NotEnoughReviews
-            }
```

Comment:
I think we should still keep this for `NoAnalysisPresent`, that's still true, we can remove the inner block for `isInitialAnalysis` as that's now being replaced by `notEnoughReviews`